include/emds/emds.h

```diff
@@ -56,14 +74,16 @@ struct emds_dynamic_entry {
  *            other value.
  * @param _data Data pointer to be stored at emergency data store.
  * @param _len Length of data to be stored at emergency data store.
+ * @param _status Pointer to EMDS entry status.
  *
  * This creates a variable _name prepended by emds_.
  */
-#define EMDS_STATIC_ENTRY_DEFINE(_name, _id, _data, _len)                      \
+#define EMDS_STATIC_ENTRY_DEFINE(_name, _id, _data, _len, _status)             \
```

Comment:
Need to be updated in rpl.c file in NCS.

Reply:
Yes you are right. I think that we must unfortunatley as i mentioned in another comment